instrumentation/github.com/gorilla/mux/otelmux/mux_test.go

```diff
@@ -190,3 +187,18 @@ func TestFilter(t *testing.T) {
 	assert.Equal(t, 1, calledHealth, "failed to run test")
 	assert.Equal(t, 1, calledTest, "failed to run test")
 }
+
+func TestRecordingResponseWriterHijack(t *testing.T) {
```

Comment:
Those two tests need to be fixed (so does the linter).
Also, we should be testing calling `Hijack()` when hijacker is implemented.